tests/components/myuplink/test_switch.py

```diff
@@ -19,6 +19,11 @@
 
 ENTITY_ID = "switch.f730_cu_3x400v_temporary_lux"
 ENTITY_FRIENDLY_NAME = "F730 CU 3x400V Tempo­rary lux"
+ENTITY_STATE = STATE_OFF
+ENTITY_STATE_ATTRIBUTES = {
```

Comment:
If it's only used once we don't need a constant.

Reply:
I know, but in next iteration (new PR) I plan to use parameterize to test more sensors based on parameter_id (unit_of_measurement) and override categories.

Reply:
Please change it then in that case. We should not add code prematurely.